eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```diff
@@ -53,6 +53,13 @@ services:
         factory_method: buildStorageEngine
         public: false
 
+    ezpublish.spi.search_handler:
+        alias: ezpublish.spi.search_engine
```

Comment:
Do we want it private? The `ezpublish.api.persistence_handler` is not private either.